README.md

```diff
@@ -19,7 +20,44 @@ How to use:
 - Create or open a project
 - Run menu Example > Example Action, it should display a list of classes
 
+## Install UML Metamodel to Eclipse 
+
+Viatra engine requires the UML Metamodel to be able to define queries and execute transformations. Hence it is necessary to install UML Metamodel to Eclipse if someone would like to use/extend the _MD_plugin_skeleton_ project. On this page, we propose a generic workflow to install UML Metamodel as a _p2 repository_.
```

Comment:
Clarification: To define and compile queries in the VQL language, VIATRA requires the UML metamodel to be installed. The engine/other runtime components do not rely on VIATRA being installed in the host Eclipse.